Update Zero Networks Audit Parser

dicolanl opened the pull request:
Change(s):
   - Update ZN Audit Parsers

   Reason for Change(s):
   - Added new audit types

v-maheshbh commented:
Hi @dicolanl 

Kindly repackage the solution using the V3 tool to ensure it reflects the latest changes in the main template. - [https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V3/README.…](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fblob%2Fmaster%2FTools%2FCreate-Azure-Sentinel-Solution%2FV3%2FREADME.md&data=05%7C02%7Cv-maheshbh%40microsoft.com%7C56dd8cc0538b4296f3a308de415218f2%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639020018067394720%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=06ArtwcH3fhfywwyQN3pXxUaThHyfZg%2FRgsey7qAvik%3D&reserved=0).

Thanks!

dicolanl commented:
@v-maheshbh was OOF will work on this soon

dicolanl commented:
@v-maheshbh 
Done

v-maheshbh commented:
Hi @dicolanl 
Manual changes were made in the mainTemplate, which is not the proper or recommended approach. We need to troubleshoot why the uri parameter was not added during solution packaging and ensure the packaging process includes all required parameters.

Thanks!

v-maheshbh commented:
Hi @dicolanl 

<img width="381" height="152" alt="image" src="https://github.com/user-attachments/assets/1ea07c11-cda0-4a71-94bd-4ccf44dc6a77" />

Kindly remove the _MACOX folder and update the release notes with the correct date.

Thanks!

dicolanl commented:
@v-maheshbh 
Added new push connector, changed the pull connector to use new CCP.
updated parser and release notes
update template and package.
Tested in azure and data connectors work as does parser.

v-maheshbh commented:
Hi @dicolanl 
There is API version mismatch between the main template inside the ZIP package and the one outside; kindly verify and ensure both are consistent.

<img width="1092" height="208" alt="image" src="https://github.com/user-attachments/assets/49aca4f8-afe2-42e0-9070-a01e9d6069af" />

 kindly attach testing screenshots of both CCF connectors in a connected state for reference.

Thanks!

v-maheshbh commented:
Hi @dicolanl 

The below error was encountered while testing the parser, as the table schema was not available for validation. Kindly attach the relevant logs

<img width="476" height="287" alt="image" src="https://github.com/user-attachments/assets/95eb0120-a408-4726-aeaf-addf1ca4b18e" />

Thanks!
